samples/Basic_DateTime/app/application.cpp

```diff
@@ -0,0 +1,119 @@
+/**	Basic_DateTime sample application
+	Author: Brian Walton (brian@riban.co.uk)
+	Date: 2018-12-10
+	Provides serial interface, accepting Unix timestamp
+	Prints each type of DateTime::format option
+*/
+#include <user_config.h>
+#include <SmingCore/SmingCore.h>
+
+time_t g_timestamp = 0;
```

Comment:
Use camelCase. For this variable and the one below.

Reply:
Fixed